docs/source/contributing.rst

```diff
@@ -183,57 +163,55 @@ Recall our folder structure diagrom from earlier. This time, we're focusing just
 	        ├── SmokeTests.py
 	        └── start_tests.py
 
-The folder and files of interest are:
+The items of interest are:
 
 - ``BaseTest.py``
```

Comment:
This uses different notation than `Kordac()` above. Doesn't need to be fixed unless you prefer consistency.